index.html

```diff
@@ -10,392 +10,422 @@
+          <div class="diagram-detail -last">
+            <div class="diagram-detail__card">
+              <!--transparent borders-->
+              <div class="icon -borders"></div>
+              <!--background-color-->
+              <div class="icon -color">
+                <!--img-->
+                <img src="./public/img/icon-complexity.png" alt="">
+              </div>
+              <h4 class="text--subheading">Complexity</h4>
+              <p class="diagram-detail__card__content">“Taking 15 medications every day is tough. Especially since they all have different instructions. How am I supposed to keep track of everything?”</p>
             </div>
-            <p class="text -small -color-accent -caps -bold">Complexity</p>
-            <p class="text -small content">“Taking 15 medications every day is tough. Especially since they all have different instructions. How am I supposed to keep track of everything?”</p>
           </div>
         </div>
       </section>
 
 
-      <!--QUESTION SECTION-->
+        <!--QUESTION SECTION-->
       <section class="question">
-        <div class="centered">
-          <p class="text -x-small -color-dark -caps -wide-spacing -bold">The Question</p>
-          <p class="text -large -serif -color-dark">How can technology help patients deal with these factors so they adhere more accurately and consistently?</p>
-        </div>
-        <div class="narrow-padding">
-          <div class="phone"></div>
-          <p class="text -small -color-dark">
-          Access was our primary concern. Smartphone adoption is growing but 35% of the U.S. population still doesn’t own one.<cite><a href="">4</a></cite> Reliance on personal computers is also problematic, especially among disadvantaged patients.</p>
-          <p class="text -small -color-dark">We narrowed our exploration to technology that was pervasive and familiar. Nearly 9 out of 10 American adults use a phone capable of texting.<cite><a href="">5</a></cite> Furthermore, adherence studies using text reminders show promising results.<cite><a href="">6</a></cite> Texting shows the potential to address difficult issues around the world and across verticals. In Libya, the development of the world’s first SMS voter registration system led to 1.5 million voter registrations.<cite><a href="">7</a></cite></p>
-          <p class="text -small -color-dark">Could texts be used not only to send reminders but also to build an ongoing two-way dialogue with a trusted health partner? Would this conversation help improve medication adherence?</p>
+        <div class="content-wrap">
+          <!--Question heading-->
+          <div class="heading-container">
+            <h2 class="text -x-small -color-dark -caps -wide-spacing -bold">The Question</h2>
+            <h3 class="text -large -serif -color-dark">How can technology help patients deal with these factors so they adhere more accurately and consistently?</h3>
```

Comment:
This should probably be a `<p>`. "The Question" is the section's heading. "How can technology..." is not really a subheading that I can tell. Font style doesn't need to dictate semantics.